app/addons/documents/jumptodoc.react.jsx

```diff
@@ -0,0 +1,62 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  '../../app',
+  '../../core/api',
+  'react',
+  'react-dom',
+  'react-select',
+  'lodash'
+], (app, FauxtonAPI, React, ReactDOM, ReactSelect, {debounce}) => {
+
+  const navToDoc = ({value: docId}) => {
+    var url = FauxtonAPI.urls('document', 'app', app.utils.safeURLName(this.props.database.id), app.utils.safeURLName(docId) );
+    FauxtonAPI.navigate(url, {trigger: true});
+  };
+
+  const JumpToDoc = ({database, allDocs}) => {
+    const options = allDocs.map(doc => {
+      return {
+        value: doc.get('_id'),
+        label: doc.get('_id')
+      };
+    });
+    return (
```

Comment:
I cannot find the option list. At this stage I'm not sure its worth making reusable until we have a reason to reuse it.